WordPressAuthenticator/Unified Auth/View Related/Get Started/GetStartedViewController.swift

```diff
@@ -124,8 +124,12 @@ class GetStartedViewController: LoginViewController, NUXKeyboardResponder {
     }()
 
     private var showsContinueButtonAtTheBottom: Bool {
-        screenMode == .signInUsingSiteCredentials ||
+        switch screenMode {
+        case .signInUsingSiteCredentials:
             configuration.enableSocialLogin == false
+        case .signInUsingWordPressComOrSocialAccounts:
+            configuration.enableSocialLogin == false
+        }
```

Comment:
Can we get rid of the switch since both cases are the same?

```suggestion
        configuration.enableSocialLogin == false
```

Reply:
We can also remove this bool and update the code as suggested below.

Reply:
Good point. Removed the switch case in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/849/commits/faa401bdfd76c7fdd1f1e72fc70c6a3a0df36660

Reply:
> We can also remove this bool and update the code as suggested below.

Answered in https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/849#discussion_r1557446892